e2e/src/stablex.rs

```diff
@@ -50,7 +50,7 @@ pub fn setup_stablex(
     for token in &tokens {
         instance
             .add_token(token.address())
-            .gas(MAX_GAS.into())
+            .gas(MAX_GAS.into()) // Why do we specify gas here and nowhere else? wasn't this already set as a default?
```

Comment:
@fleupold - I think this could be removed and it wouldn't change anything.

Reply:
I'm not sure, but I think this was added because the automatic gas estimation was not working correctly for contract deployment with Ganache. I'm not super confident that this is the reason though.

I think that if you remove it and we get the green check, than we should take it out!

Reply:
Notice that this is set at the beginning of the file (which is not included in the diff)

Here: https://github.com/gnosis/dex-services/blob/0d2a4ed486ba852f3a29d2e682ab3d5f1210008d/e2e/src/stablex.rs#L23

Reply:
yes, feel free to remove.